ckan/model/__init__.py

```diff
@@ -221,7 +228,10 @@ def create_db(self) -> None:
         i.e. the same as init_db APART from when running tests, when init_db
         has shortcuts.
         '''
-        self.metadata.create_all(bind=self.metadata.bind)
+        if not meta.engine:
+            raise CkanConfigurationException("Model is not initialized")
```

Comment:
Same here. What should we do at this point if the Model is not initialized? What's the cause of it? Some pointers to the user would be great.